schedule/yast/autoyast/autoyast_mig_spvm.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: autoyast_mig_spvm
+description: >
+  Migration autoYaST installation and do prepare for migration standalone
```

Comment:
'standalone' is for ha test and not for other installation, so please remove it.

Reply:
Updated.